mgmtsystem_nonconformity/reports/mgmtsystem_nonconformity_report.py

```diff
@@ -0,0 +1,133 @@
+# -*- coding: utf-8 -*-
+
+from openerp import fields, models,  _
+from openerp import tools
+
+
+class MgmtsystemtNonConformityReport(models.Model):
+    """Management System Non Conformities Report."""
+
+    _name = "mgmtsystem.nonconformity.report"
+    _auto = False
+    _description = "Management System Non Conformity Report"
+    _rec_name = 'id'
+
+    _STATES = [
+        ('draft', _('Draft')),
+        ('analysis', _('Analysis')),
+        ('pending', _('Pending Approval')),
+        ('open', _('In Progress')),
+        ('done', _('Closed')),
+        ('cancel', _('Cancelled')),
+    ]
+
+    # Compute data
+    number_of_nonconformities = fields.Integer(
+        '# of nonconformities', readonly=True)
+    age = fields.Integer('Age', readonly=True)
+    number_of_days_to_analyse = fields.Integer(
+        '# of days to analyse', readonly=True)
+    number_of_days_to_plan = fields.Integer('# of days to plan', readonly=True)
+    number_of_days_to_execute = fields.Integer(
+        '# of days to execute', readonly=True)
+    number_of_days_to_close = fields.Integer(
+        '# of days to close', readonly=True)
+
+    # Grouping view
+
+    date = fields.Datetime('Date', readonly=True, select=True)
+    create_date = fields.Datetime('Create Date', readonly=True, select=True)
+    analysis_date = fields.Datetime('Analysis Date', readonly=True,
+                                    select=True)
+    actions_date = fields.Datetime(
+        'Action Plan evaluation Date', readonly=True, select=True)
+    evaluation_date = fields.Datetime(
+        'Efficiency evaluation Date', readonly=True, select=True)
+    closing_date = fields.Datetime('Closing Date', readonly=True, select=True)
+    cancel_date = fields.Datetime('Cancel Date', readonly=True, select=True)
+    partner_id = fields.Many2one('res.partner', 'Partner', required=True)
+    responsible_user_id = fields.Many2one(
+        'res.users', 'Responsible', readonly=True)
+    manager_user_id = fields.Many2one('res.users', 'Manager', readonly=True)
+    author_user_id = fields.Many2one('res.users', 'Filled by', readonly=True)
+    analysis_user_id = fields.Many2one('res.users', 'Analysis Evaluator',
+                                       readonly=True)
+    evaluation_user_id = fields.Many2one('res.users', 'Efficiency Evaluator',
+                                         readonly=True)
+    actions_user_id = fields.Many2one('res.users', 'Plan Evaluator',
+                                      readonly=True)
+    cause_ids = fields.Many2many(
+        'mgmtsystem.nonconformity.cause',
+        'mgmtsystem_nonconformity_cause_rel',
+        'nonconformity_id',
+        'cause_id',
+        'Cause',
+    )
+    severity_id = fields.Many2one(
+        'mgmtsystem.nonconformity.severity',
+        'Severity', readonly=True,
+    )
+    origin_ids = fields.Many2many(
```

Comment:
@dreispt My pivot view doesn't work when selecting a many2many field in the init method. Please i need help